docker/Dockerfile

```diff
@@ -116,6 +116,8 @@ RUN --mount=type=cache,id=conda_pkgs,target=/opt/conda/pkgs,sharing=locked \
     conda config --add pkgs_dirs /opt/conda/pkgs &&\
     # Install mamba, boa and git here. Conda build breaks with other git installs
     /opt/conda/bin/mamba install -y -n base -c conda-forge "boa" "git >=2.35.3" "python=${PYTHON_VER}" "tini=0.19" &&\
+    # CVE-2018-20225 for the base pip, not the env one
+    python -m pip uninstall -y pip && \
```

Comment:
Can you move this to the `runtime` stage, which I believe is the final stage for the images that we upload to NGC

Reply:
@dagardner-nv done. PTAL